src/framework/standard/structures/check.rs

```diff
@@ -1,9 +1,10 @@
-use std::fmt::Debug;
-use std::fmt;
-use crate::model::channel::Message;
 use crate::client::Context;
 use crate::framework::standard::{Args, CommandOptions};
+use crate::model::channel::Message;
 use futures::future::BoxFuture;
+use std::error::Error;
+use std::fmt;
+use std::fmt::{Debug, Display};
```

Comment:
Import format nitpick.
```suggestion
use std::fmt::{self, Debug, Display};
```